examples/tfengine/modules/team.hcl

```diff
@@ -468,6 +468,17 @@ template "additional_iam_members" {
           ]
         }
       }]
+      "service_account" = [{
+        resource_ids = [
+          "runner@{{.prefix}}-{{.env}}-apps.iam.gserviceaccount.com",
+        ]
+        bindings = {
+          "roles/iam.serviceAccountKeyAdmin" = [
+            "serviceAccount:runner@{{.prefix}}-{{.env}}-apps.iam.gserviceaccount.com"
```

Comment:
Any other entity we can use to assign this role? Right now, we are assigning a service account role to the service account itself.